frontend/app/routes-flow/apply-flow.ts

```diff
@@ -85,11 +85,18 @@ const communicationPreferencesStateSchema = z.object({
  * Schema for communication reference.
  */
 const demographicsPart2StateSchema = z.object({
-  gender: z.string().min(1),
+  gender: z.string().trim().optional(),
   otherGender: z.string().min(1).optional(),
-  mouthPainType: z.string().min(1),
-  lastDentalVisitType: z.string().min(1),
-  avoidedDentalCareDueToCost: z.string().min(1),
+  mouthPainType: z.string().trim().optional(),
+  lastDentalVisitType: z.string().trim().optional(),
+  avoidedDentalCareDueToCost: z.string().trim().optional(),
+});
+
+const demographicsPart1StateSchema = z.object({
+  firstNations: z.string().trim().optional(),
+  firstNationsEnum: z.string().trim().optional(),
+  bornType: z.string().trim().optional(),
+  disabilityType: z.string().trim().optional(),
 });
```

Comment:
These validations will allow `'      '` (ie: a string of all whitespace). Is this your intent?